src/test/java/org/opensearch/security/SecurityAdminTests.java

```diff
@@ -59,15 +64,89 @@ public void testSecurityAdmin() throws Exception {
         int returnCode  = SecurityAdmin.execute(argsAsList.toArray(new String[0]));
         Assert.assertEquals(0, returnCode);
         
-        RestHelper rh = nonSslRestHelper();
-        HttpResponse res;
-        
-        Assert.assertEquals(HttpStatus.SC_OK, (res = rh.executeGetRequest("_opendistro/_security/health?pretty")).getStatusCode());
+        RestHelper rh = restHelper();
+
+        Assert.assertEquals(HttpStatus.SC_OK, (rh.executeGetRequest("_opendistro/_security/health?pretty")).getStatusCode());
     }
-    
+
+    @Test
+    public void testSgAdminInvalidCert() throws Exception {
```

Comment:
Please rename the method name to `testSecurityAdminInvalidCert`

Reply:
done